spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
```

Comment:
Drop the === at the end of the section header.  Only the first === at the beginning are required.